css/style.css

```diff
@@ -3,3 +3,26 @@
     margin-top: 2em;
     margin-bottom: 2em;
 }
+
+.append input[type=range] + input[type=text] {
+    border-radius: 0px;
+    -webkit-border-radius: 0px;
```

Comment:
why do you need -webkit attributes? this extension is only supported on latest versions of Chrome, so border-radius alone should be fine.